### PR TITLE
Update golangci-lint

### PR DESCRIPTION
**Pull request by zoncoen**
Update `golangci-lint` version and fix lint errors.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -131,7 +135,7 @@ gen/mock: $(GOTYPENAMES) $(MOCKGEN)
 		echo "generate mock for $$file"; \
 		dstfile=$$(dirname $$file)/$$(basename $${file%.pb.go})_mock.go; \
 		gotypenames --filename $$file --only-exported --types interface | xargs -ISTRUCT -L1 -P8 mockgen -source $$file -package $$package -self_package $(GEN_PB_DIR)/$$package -destination $$dstfile; \
-		sed -i '' -e '/\/\/ Source: /d' $$dstfile || (echo "failed to delete generated marker about source path ( Source: /path/to/name.pb.go )"); \
+		sed -i -e '/\/\/ Source: /d' $$dstfile || (echo "failed to delete generated marker about source path ( Source: /path/to/name.pb.go )"); \
```
Comment (goccy) [inline Makefile:138]: Maybe...
I would think if `''` removed, temporary files are generated at executed `sed -i` on macOS.

Comment (zoncoen) [reply Makefile:138]: Hmm... I got an error on Linux 😇

Comment (zoncoen) [reply Makefile]: Use `perl` command instead of `sed` to avoid platform-dependent difference.

**Review by goccy — APPROVED** [review]
LGTM !!!!

**Pull request merged**